src/Elmish/Component.purs

```diff
@@ -364,10 +344,10 @@ wrapWithLocalState' name mkDef =
 -- | instance. This means that, on one hand, we cannot use the same React class
 -- | for every instantiation, because this may create conflicts, where one
 -- | Elmish component replaces another in the DOM, but they look like the same
--- | component to React, which makes it reuse state, which leads to chaos. On
--- | the other hand, we cannot create a fresh class on every render, because
--- | then React will see it as a new component every time, and will reset its
--- | state every time.
+-- | component to React, which makes it reuse state, which leads to breaking
+-- | type safety. On the other hand, we cannot create a fresh class on every
+-- | render, because then React will see it as a new component every time, and
+-- | will reset its state every time.
 -- |
 -- | This means that we need some way of figuring out whether it needs to be
 -- | logically "same" component or "different", but there is no way to get that
```

Comment:
```suggestion
-- | logically the "same" component or "different", but there is no way to get that
```

Reply:
Oh, thank you very much. Russian doesn't have these bloody articles, and you can probably tell that I still haven't developed an intuition for when and how to use them :-(

Reply:
No worries! To make things more complicated, things like documentation often omit them in English, too.